src/components/NoLocalOrganisations/NoLocalOrganisations.module.scss

```diff
@@ -0,0 +1,21 @@
+@use '@assets/global.scss' as g;
+
+.container {
+  position: relative;
+  color: g.$primary-dark-blue;
+  height: 280px;
+  width: 800px;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+}
+
+.contactUs {
+  text-decoration: underline;
+}
+
+.container svg {
+  position: absolute;
+  left: 21rem;
+  top: 10rem;
+}
```

Comment:
Super dodgy CSS I know, but I couldn't find another way to move the heart logo to the centre of the container. Tried align-items, justify-content, text-align... all that good stuff. If you can think of a better way to do this, I'm all ears

Reply:
quick note on the above- I think the reason why I found it difficult to shift the logo to the centre is because the svg code itself places the logo to the left, if that makes sense. I created the HandHeart svg by just copy & pasting some of the code from `src/assets/logo/LogoBlue.tsx`, where the section of the logo I copied is shifted to the left, as such:

![image](https://github.com/CapgeminiInventUK/donate-to-educate-frontend/assets/10779091/d935f4a0-9936-489b-8969-23e96f59d965)